webapp/views.py

```diff
@@ -48,7 +48,7 @@
 )
 
 
-def _build_mirror_list():
+def _build_mirror_list(all=False):
```

Comment:
How about doing it the other way around? Something like `nearby=False`, or `use_geoip=False`?

Reply:
Good call, updated to `local` as discussed.